Library/Formula/dirmngr.rb

```diff
@@ -18,6 +18,11 @@ class Dirmngr < Formula
   depends_on "libksba"
   depends_on "pth"
 
+  patch :p0 do
+    url "http://bugs.g10code.com/gnupg/file419/dirmngr-pth-fix.patch"
+    sha256 "0efbcf1e44177b3546fe318761c3386a11310a01c58a170ef60df366e5160beb"
+  end
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!